circle.yml

```diff
@@ -0,0 +1,34 @@
+machine:
+  environment:
+    COVALENCE_CMD: "docker run -v /home/ubuntu/.aws:/root/.aws -v /var/run/docker.sock:/var/run/docker.sock -v /home/ubuntu/dockerfile-covalence/uat:/home/ubuntu/dockerfile-covalence/uat -w /home/ubuntu/dockerfile-covalence/uat"
```

Comment:
hmm, i kind of prefer passing through the env values rather than the built in circle CI AWS file. The volume mount can also be easily missed if someone attempted to run the container with a different non-root user. since I'd assume `gosu` and aws-cli wouldn't be looking at `/root/.aws`.